diff --git a/libretro/Makefile b/libretro/Makefile
index a96ac85..4b92822 100644
--- a/libretro/Makefile
+++ b/libretro/Makefile
@@ -1,4 +1,3 @@
-DEBUG = 0
 HAVE_EXCEPTIONS = 0
 HAVE_STRINGS_H = 1
 
@@ -164,6 +163,15 @@ else ifeq ($(platform), classic_armv8_a35)
 	    LDFLAGS += -static-libgcc -static-libstdc++
 	  endif
 	endif
+
+else ifeq ($(platform), goadvance)
+	TARGET := $(TARGET_NAME)_libretro.so
+	fpic = -fPIC
+	SHARED := -shared -Wl,--version-script=link.T -Wl,--no-undefined
+	CXXFLAGS += -march=armv8-a+crc+simd -mtune=cortex-a35 -mcpu=cortex-a35 -Ofast -pipe -fno-plt -fdata-sections -ffunction-sections -Wl,--gc-sections -fno-stack-protector -fno-ident -fomit-frame-pointer -fmerge-all-constants -fno-math-errno -Wl,--gc-sections -falign-functions=1 -falign-jumps=1 -falign-loops=1 -fno-unwind-tables -fno-asynchronous-unwind-tables -fno-unroll-loops -fmerge-all-constants -frename-registers -funsafe-math-optimizations -ftree-vectorize -flto=16 -fuse-linker-plugin
+	CXXFLAGS += -flto=16 -fprofile-use -fprofile-dir=/tmp
+   CFLAGS += $(CXXFLAGS)
+	LDFLAGS += -mcpu=cortex-a35 -flto=16 -fuse-linker-plugin -fprofile-use -fprofile-dir=/tmp
+
 #######################################
    
 # iOS
